### PR TITLE
zh-cn: init the translation of "Document an HTTP header"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

init the translation of "Document an HTTP header"

### CODE REVIEW

File: files/zh-cn/mdn/writing_guidelines/howto/document_an_http_header/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- 转到现有的具有相同类别的标头参考页。
+
+## 步骤 3——创建 HTTP 标头页面
+
+- 所有标头页面都位于此目录下：[/docs/Web/HTTP/Headers/](/zh-CN/docs/Web/HTTP/Headers)
+- 要创建新页面，请参阅我们的[如何创建页面](/zh-CN/docs/MDN/Writing_guidelines/Howto/Creating_moving_deleting)指南中的说明。
+
+## 步骤 4——撰写内容
+
+- 从我们的 [HTTP 标头模板页面](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Page_types#http_header_reference_page)开始，或者使用你在步骤 2 中找到的现有 HTTP 标头文档之一复制的结构。由你自己决定。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/writing_guidelines/howto/document_an_http_header/index.md:33]:
```suggestion
- 从我们的 [HTTP 标头模板页面](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Page_types#http_标头参考页)开始，或者使用你从步骤 2 中找到的现有 HTTP 标头文档结构复制之一。由你自己决定。
```

**Review by jasonren0403 — APPROVED** [review]
One minor change

File: files/zh-cn/mdn/writing_guidelines/howto/document_an_http_header/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/mdn/writing_guidelines/howto/document_an_http_header/index.md:33]: How about this?

```suggestion
- 从我们的 [HTTP 标头模板页面](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Page_types#http_标头参考页)开始，或者使用你从步骤 2 中找到的现有 HTTP 标头文档中的一个复制的结构。由你自己决定。
```

Comment (jasonren0403) [reply files/zh-cn/mdn/writing_guidelines/howto/document_an_http_header/index.md:33]: > How about this?

Better 👍

**Pull request merged**